conda_smithy/templates/azure-pipelines-win.yml.tmpl

```diff
@@ -75,8 +70,14 @@ jobs:
           set "EXTRA_CB_OPTIONS=%EXTRA_CB_OPTIONS% --no-test"
         )
         {%- endif %}
-        {%- if build_with_mambabuild %}
+        {%- if conda_build_tool == "mambabuild" %}
         conda.exe mambabuild "{{ recipe_dir }}" -m .ci_support\%CONFIG%.yaml --suppress-variables %EXTRA_CB_OPTIONS%
+        {%- elif conda_build_tool == "conda-build+conda-libmamba-solver" %}
+        set "CONDA_SOLVER=libmamba"
+        conda.exe build "{{ recipe_dir }}" -m .ci_support\%CONFIG%.yaml --suppress-variables %EXTRA_CB_OPTIONS%
```

Comment:
Would it makes sense to either specify this by the `--solver` flag or better yet add this to the `.condarc`

Reply:
`conda-build` doesn't have the `--solver` flag so the other alternative is writing to `.condarc`, but that will have global impact on the other uses of `conda` so I initially wanted to constrain the scope as much as possible (in case something breaks).

I am happy to revisit that in the future as we debug it step by step.